pom.xml

```diff
@@ -54,7 +54,7 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-
```

Comment:
This is also fixed in another PR it may have a conflict later

Reply:
I needed to fix it because I had to lift the application, thanks for your observation